lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/document_builder_api_common/document_builder.rb

```diff
@@ -7,18 +7,41 @@ module DocumentBuilder
     include PageObject
 
     DEFAULT_BUILDER_FILE_NAME = 'SampleText.docx'.freeze
+    DEFAULT_BUILDER_DOCX_FILE_NAME = 'Sample.docx'.freeze
+    DEFAULT_BUILDER_XLSX_FILE_NAME = 'Sample.xlsx'.freeze
 
     # actions
     link(:generate_document, xpath: '//*[@id="generateButton"]')
+    link(:create_document, xpath: '//*[@id="createDocx"]')
+    link(:create_spreadsheet, xpath: '//*[@id="createXlsx"]')
 
-    def builder_works?
+    def builder_download
       generate_document_element.click
-      path_to_downloaded_file = @instance.webdriver.download_directory + '/' + DEFAULT_BUILDER_FILE_NAME
-      OnlyofficeFileHelper::FileHelper.wait_file_to_download(path_to_downloaded_file)
-      file_size = File.size(path_to_downloaded_file)
+      @path_to_downloaded_file = @instance.webdriver.download_directory + '/' + DEFAULT_BUILDER_FILE_NAME
```

Comment:
Do not store `@path_to_download_file` as class variable, this'll be messed up while run parallel
save it as `path_to_downloaded_file` and in `builder_works?` method use returned value like
```
path_to_file = builder_download
file_size = File.size(path_to_file)
```